### ISSUE TITLE
Cannot edit names on SketchList or CollectionList using keyboard controls

### ISSUE THREAD

**Issue by lindapaiste** [processing/p5.js-web-editor#2383]
### What is your operating system?

None

### Actual Behavior

The name turns into an input, but immediately closes.

This is because the "Enter" keyboard event which was used to select "Rename" is still being processed, and "Enter" will submit the form.

### Expected Behavior

The name should turn into an input and stay that way until you press "Enter" again.

### Steps:
1. Go to your `/sketches` page. 
2. Click somewhere on the page to focus.
3. Press "Tab" until you focus on the dropdown at the end of the table row.
![image](https://github.com/processing/p5.js-web-editor/assets/28965286/8b6b35d3-fe4c-481f-ac06-08de4d425651)
4. Press "Enter" to open the dropdown.
5. Press "Tab" to focus on "Rename".
![image](https://github.com/processing/p5.js-web-editor/assets/28965286/c6a0bdf1-a1d3-47c7-9339-a1698e5f62ad)
6. Press "Enter" to select "Rename".
7. Try to edit the name...you can't.

---

I came across this issue while working on #2382 and it will be fixed in that PR.  We could also implement a more immediate fix for just this issue if we deem it high-priority.

**Comment by gajanandh** [comment]
hi @lindapaiste  ,I would like to fix this issue can you please assign it to me..?

**Comment by Ankush263** [comment]
Hey @lindapaiste, I solved this issue locally, should I raise a PR??
